src/app/tests/suites/certification/Test_TC_LVL_2_1.yaml

```diff
@@ -23,116 +23,140 @@ tests:
       cluster: "DelayCommands"
       command: "WaitForCommissionee"
 
-    - label: "reads current Level attribute from DUT"
+    #Disabled due to issue - #11670
```

Comment:
That does not seem right.  That issue (or at least its screenshot) is about incorrect XML for the thermostat cluster, but this is the level control cluster.  If the XML here is also wrong, please file an issue on that and reference it here?

Reply:
i will update with proper issue ID